<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameDefinitionForm</class>
 <widget class="QWidget" name="GameDefinitionForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>673</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="skyrimSeLogoLabel">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>Resources/skyrimse2.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="skyrimSeLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>SKYRIM SE</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="skyrimSeFilesLineLabel">
             <property name="text">
              <string>%file line%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="applyPathSkyrimSeToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="selectPathToSkyrimSeToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBarSkyrimSe">
           <property name="value">
            <number>99</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="format">
            <string>%p%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="skyrimLeLogoLabel">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>Resources/skyrimle.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="skyrimLeLabel">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>SKYRIM LE</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="skyrimLeFilesLineLabel">
             <property name="text">
              <string>%file line%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="applyPathSkyrimLeToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="selectPathToSkyrimLeToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBarSkyrimLe">
           <property name="value">
            <number>99</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="format">
            <string>%p%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="F4LogoLabel">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>Resources/fallout4.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="F4Label">
             <property name="font">
              <font>
               <family>Yu Gothic UI</family>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="text">
              <string>FALLOUT 4</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="F4FilesLinelabel">
             <property name="text">
              <string>%file line%</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="applyPathF4ToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="selectPathToF4ToolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBarF4">
           <property name="value">
            <number>99</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
           <property name="format">
            <string>%p%</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QComboBox" name="selectScanGameComboBox">
         <item>
          <property name="text">
           <string>SKYRIM SE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>SKYRIM LE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FALLOUT 4</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="scanPushButton">
         <property name="text">
          <string>Scan</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okPushButton">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quitPushButton">
         <property name="font">
          <font>
           <family>Yu Gothic UI</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
